CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ development)_
 
 ## [Unreleased]
 
-**Expected next version:** 2.0.0
+## [2.0.0] - 20-07.2021
```

Comment:
Ok but now the date is a bit strange…

- format should be either 20.07.2021 or 2021-07-20 IMHO (I’d prefer 2021-07-20)
- today is the 19th – do you want to wait with the release until tomorrow?